plugins/spark/v3.5/spark/build.gradle.kts

```diff
@@ -78,15 +57,9 @@ dependencies {
     exclude("org.apache.commons", "*")
   }
 
-  implementation("org.apache.iceberg:iceberg-core:${icebergVersion}")
```

Comment:
cc @dimas-b, it doesn't depend on iceberg-core anymore, not leaking of transitive dependencies.